src/display/api.js

```diff
@@ -805,6 +805,24 @@ class PDFDocumentProxy {
     return this._transport.getMetadata();
   }
 
+  /**
+   * @typedef {Object} MarkInfo
+   * Properties correspond to Table 321 of the PDF 32000-1:2008 spec. Values
+   * are null if not defined by the PDF.
+   * @property {boolean | null} Marked
```

Comment:
I looked at https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf#page=564&zoom=auto,-215,328 and all those fields should have the default value `false`, so I think we should implement that as well. Moreover, then we can always have a boolean here, which also seems like a better API and more logical when looking at the unit test. Finally, I think it's also necessary for https://github.com/mozilla/pdf.js/pull/12525/files#diff-e24e4bd32090b0c400a70f02d1b844df95c9bbe62cd0b280f4397e9f4194fe44R1352 to work properly, since we'd now be reporting `null` if there is `MarkInfo` without a `Marked` flag.